tests/issue/109/unit/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
+
+include("../../../cmake/modules/hunter_boost_component_b2_args.cmake")
+project(TestIssue109Unit)
+
+
+set(component "foo")
+set(cmakeArgs "FOO_OPT=1")
+
```

Comment:
Also what about test with multiple options, is this allowed? What it must looks like `-s FOO_OPT=1 BOO_OPT=2` or `-s FOO_OPT=1 -s BOO_OPT=2`?